uint8 index
uint8 status # 0:未到達 or 1:到達